JitsiConference.js

```diff
@@ -2392,9 +2413,20 @@ JitsiConference.prototype._startP2PSession = function(peerJid) {
     this.p2pJingleSession.initialize(true /* initiator */, this.room, this.rtc);
 
     logger.info('Starting CallStats for P2P connection...');
+
+    let remoteID = Strophe.getResourceFromJid(this.p2pJingleSession.peerjid);
+
+    if (this.options.config.enableStatsID) {
+        const participant = this.participants[remoteID];
+
+        if (participant) {
+            remoteID = participant.getStatsId() || remoteID;
+        }
+    }
```

Comment:
I'm not gonna be strict on this: do you see a simple way to remove the duplicated logic for getting the remoteID?

Reply:
Well will not make it any better ...
        if (participant && participant.getStatsId()) {
            remoteID = participant.getStatsId();
        }
Do you think this is better?